domains/krunch.json

```
@@ -1,11 +1,12 @@
 {
   "description": "Profile Page",
-  "repo": "https://github.com/RadioactivePotato/krunch-is-a-dev",
+  "repo": "https://github.com/RadioactivePotato/site",
```

```suggestion
  "repo": "https://github.com/RadioactivePotato/site"
```
This comma should not be here.